### PR TITLE
[FSPE-6932] Add support of removing Tooltip delay when navigating between close Tooltips

### PR DESCRIPTION
**Pull request by janibolkvadze**
### Commit summary

- [x] Added functionality of disabling animation delay when the user navigates quickly from one Tooltip trigger button to another;
- [x] Reduced Tooltip delay on Show from 500 to 300;
- [x] Changed Tooltip hide behavior when the Tooltip trigger button is clicked
- [x] Show tooltip when using Tab or Shift + Tab

@dani-cl-madrid @SergioQCL @mattnickles @jllr 

https://ctl.atlassian.net/browse/FSPE-6932

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e8d75cc03eb3c50d1e37ccb2c7323554270a0267-chi-pr.enterprise.ctl.io" target="_blank"> https://e8d75cc03eb3c50d1e37ccb2c7323554270a0267-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1650/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://7379a28e9140531eba8e47e6d586ac2618753ab8-chi-pr.enterprise.ctl.io" target="_blank"> https://7379a28e9140531eba8e47e6d586ac2618753ab8-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1695/ for more details.

File: src/chi/javascript/components/tooltip.js [hunk 1]
```diff
@@ -39,21 +40,37 @@ class Tooltip extends Component {
...
       }
     });
     this._addEventHandler(this._elem, 'focus', function() {
       self._focused = true;
-      if (!self._shown) {
-        self.show();
+      self._addEventHandler(self._elem, 'keyup', function(e) {
+        let code = (e.keyCode ? e.keyCode : e.which);
+
+        if (code === 9) {
```
Comment (SergioQCL) [inline src/chi/javascript/components/tooltip.js:66]: Which key is coded with 9? Maybe a small comment will be handy here

Comment (janibolkvadze) [reply src/chi/javascript/components/tooltip.js:66]: it's `Tab` key.
Good point, what if we define a reusable constant, something like:

```
export const keys = {
  tab: 9,
  left: 37,
  up: 38,
  right: 39,
  down: 40,
  delete: 46
};
```

to be imported in keyboard controlled components?

**Review by mattnickles — APPROVED** [review]

File: src/chi/javascript/components/tooltip.js (hunk 1 above)
Comment (SergioQCL) [reply src/chi/javascript/components/tooltip.js:66]: Yes, good idea! I'd define the names in caps, though. 👍🏻

Comment (janibolkvadze) [reply src/chi/javascript/components/tooltip.js]: Done, thanks! 😉

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://deade61afc39d7017d98f951962e2689f5415eda-chi-pr.enterprise.ctl.io" target="_blank"> https://deade61afc39d7017d98f951962e2689f5415eda-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1703/ for more details.

**Review by SergioQCL — APPROVED** [review]
ACK

File: src/chi/javascript/components/tooltip.js (hunk 1 above)
Comment (SergioQCL) [reply src/chi/javascript/components/tooltip.js]: Much better! 👍🏻

**Review by jllr — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f2f2e9b34433c368d4d43bd701edd4fb79197a30-chi-pr.enterprise.ctl.io" target="_blank"> https://f2f2e9b34433c368d4d43bd701edd4fb79197a30-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1706/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://ad6f93dbd6621dce0fb20733010d2df9662edf63-chi-pr.enterprise.ctl.io" target="_blank"> https://ad6f93dbd6621dce0fb20733010d2df9662edf63-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1707/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-gd1870, https://ops.elasticbox.com/services/instances/i-sd5l8x, https://ops.elasticbox.com/services/instances/i-opm9re, https://ops.elasticbox.com/services/instances/i-fsmcf6, https://ops.elasticbox.com/services/instances/i-ey24cw

**Pull request merged**